updateFeatureFlag/main.go

```diff
@@ -31,7 +31,7 @@ func handleValidationError(err error) []utils.ValidationError {
 			Field: err.Field(),
 			Error: err.Tag(),
 		})
-		log.Println("Errors are ", errors)
+		log.Printf("Error while validating updateFeatureflag: %v", errors)
```

Comment:
would prefer if errors could come in next line else it would be like
```
Error loading env file could not find the key
some unknown error
xyz
```

Reply:
Done